marshal/oracle/listpdb.go

```diff
@@ -18,27 +18,44 @@ package oracle
 import (
 	"bufio"
 	"bytes"
+	"github.com/ercole-io/ercole-agent/v2/marshal"
+	ercutils "github.com/ercole-io/ercole/v2/utils"
+	"github.com/hashicorp/go-multierror"
 	"strings"
 
 	"github.com/ercole-io/ercole/v2/model"
 )
 
 // ListPDB returns information about pdbs extracted
 // from the listdb fetcher command output.
-func ListPDB(cmdOutput []byte) []model.OracleDatabasePluggableDatabase {
+func ListPDB(cmdOutput []byte) ([]model.OracleDatabasePluggableDatabase, error) {
 	pdbs := []model.OracleDatabasePluggableDatabase{}
 	scanner := bufio.NewScanner(bytes.NewReader(cmdOutput))
+	var merr error
 
 	for scanner.Scan() {
 		pdb := new(model.OracleDatabasePluggableDatabase)
 		line := scanner.Text()
+
+		if len(strings.TrimSpace(line)) == 0 {
+			continue
+		}
 		splitted := strings.Split(line, "|||")
+		iter := marshal.NewIter(splitted)
 
-		pdb.Name = strings.TrimSpace(splitted[0])
-		pdb.Status = strings.TrimSpace(splitted[1])
+		if pdb.Name = strings.TrimSpace(iter()); pdb.Name == " " {
+			merr = multierror.Append(merr, ercutils.NewErrorf("Pdb Name not found"))
+		}
+		if pdb.Status = strings.TrimSpace(iter()); pdb.Status == " " {
+			merr = multierror.Append(merr, ercutils.NewErrorf("Pdb Status not found"))
+		}
 		pdb.Services = []model.OracleDatabaseService{}
 
 		pdbs = append(pdbs, *pdb)
 	}
-	return pdbs
-}
+
+	if merr != nil {
+		return nil, merr
+	}
+	return pdbs, nil
+}
```

Comment:
Add a newline at the end of the file, using a linter should help!

![image](https://user-images.githubusercontent.com/1636493/148803584-90ff91ae-efe9-4d8e-adb9-b0058741e8d2.png)